BedrockNode.cpp

```diff
@@ -67,7 +63,6 @@ BedrockNode::BedrockNode(const SData& args, BedrockServer* server_)
     SINFO("BedrockNode constructor");
 }
 
-// --------------------------------------------------------------------------
```

Comment:
I removed them here because the trend has been to remove them everywhere. I don't care that much either way, I was just going for consistency.